zerver/lib/markdown/tabbed_sections.py

```diff
@@ -98,6 +98,7 @@
     "okta": "Okta",
     "onelogin": "OneLogin",
     "azuread": "AzureAD",
+    "entraid": "Microsoft Entra ID",
```

Comment:
This reminds me, Microsoft has renamed AzureAD to EntraID a while back - https://learn.microsoft.com/en-us/entra/fundamentals/new-name

So as confusing as it is, we may want to update our references to AzureAD. Though many people still reference it as AzureAD I think, so I feel it's confusing whichever way we go with this.

Reply:
We can call it EntraID/AzureAD if that feels clearest, I suppose.